es/utils/string.js

```diff
@@ -56,6 +56,16 @@ export function isHex (str) {
   return !!(str.length % 2 === 0 && str.match(/^[0-9a-f]+$/i))
 }
 
+/**
+ * Check whether a string is Aeternity address
+ * @param {string} str String to validate.
+ * @return {boolean} True if the string is valid AE address
+ */
+export function isAeAddress (str) {
```

Comment:
This only checks if the string is prefixed with 2 char string before the `_`. E.g. `ak_`

Shouldn't it also check for `Crypto.isAddressValid` ?

Reply:
Yes, you are right